client/components/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -24,9 +25,12 @@ const DateControlPickerDate = ( {
 			} ) }
 		>
 			<h2 className={ `${ BASE_CLASS_NAME }__heading` }>
-				{ translate( 'Date Range' ) }
-				<span> &#8212;</span>
-				<input id="date-example" type="date" disabled />
+				{ overlay && <Icon icon={ lock } /> }
+				<div className={ `${ BASE_CLASS_NAME }s__heading-text` }>
```

Comment:
I put this in a div to fix the horizontal flex alignment. Otherwise the span and input look weird. I tried fixing it with flex properties but that didn't look good in Firefox.